sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -122,7 +128,33 @@ bool rom_test_main(void) {
     LOG_INFO("Jitter is enabled");
   }
 
+#if !OT_IS_ENGLISH_BREAKFAST
+  // Check the otp to see if flash scramble should be enabled.
+  CHECK_DIF_OK(dif_otp_ctrl_init(
+      mmio_region_from_addr(TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR), &otp_ctrl));
+
+  uint32_t otp_val;
+  otp_val = abs_mmio_read32(
+      TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET +
+      OTP_CTRL_PARAM_CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG_OFFSET);
+
+  // very stupid, just set the entire first bank to scrambled right now
+  if (otp_val != 0) {
+    LOG_INFO("Default flash settings have been supplied through otp 0x%x",
+             otp_val);
+
+    dif_flash_ctrl_region_properties_t default_properties;
+    CHECK_DIF_OK(dif_flash_ctrl_get_default_region_properties(
+        &flash_ctrl, &default_properties));
+    default_properties.scramble_en = bitfield_field32_read(
+        otp_val, FLASH_CTRL_DEFAULT_REGION_SCRAMBLE_EN_FIELD);
+    CHECK_DIF_OK(dif_flash_ctrl_set_default_region_properties(
+        &flash_ctrl, default_properties));
+  }
+#endif
+
   if (bootstrap_requested() == kHardenedBoolTrue) {
+    LOG_INFO("boot strap requested!");
```

Comment:
```suggestion
    LOG_INFO("bootstrap requested!");
```

Reply:
will fix! i'm not actually sure I need this anymore. I originally added it to synchronize with DV, but i forgot Alex's bootstrap already applies a reset. Let me remove this altogether.

Reply:
i was actually wrong..the beginning of the bootstrap routine doesn't actually call reset.
So if I remove the software reset, the routine just times out waiting for certain register changes.
As a result, I've decided to just keep this here.

Reply:
btw we also have `OT_ADDRESSABLE_LABEL`, not sure which one you prefer but that one is definitely leaner.

Reply:
i actually don't even know what that is. let me have a look.

Reply:
SG, just wanted to mention since it looked like it could also be used for this purpose.